test/libyul/Common.h

```diff
@@ -53,4 +55,6 @@ parse(std::string const& _source, Dialect const& _dialect, langutil::ErrorList&
 Block disambiguate(std::string const& _source, bool _yul = true);
 std::string format(std::string const& _source, bool _yul = true);
 
+solidity::yul::Dialect const& getDialect(std::string const& _name, langutil::EVMVersion _evmVersion);
```

Comment:
Just `dialect`?

Reply:
Do you mean that `getDialect` is too general? We could call it `getDialectByNameAndEVMVersion`, but it does not sound quite right either.

Reply:
No, I mean the function can be called `dialect` or maybe `dialectByName`.

Reply:
Done.